models/BaseDoc.php

```diff
@@ -40,6 +44,24 @@ class BaseDoc extends BaseObject
     public $tags = [];
 
 
+    /**
+     * @param Type|null $aggregatedType
+     * @return string[]|array
```

Comment:
```suggestion
     * @return string[]
```

Reply:
Thanks. Because PR is already merged, I'll fix it in a separate commit.